core/new-gui/src/app/workspace/component/workflow-editor/workflow-editor.component.ts

```diff
@@ -79,6 +88,42 @@ export class WorkflowEditorComponent implements AfterViewInit {
     this.setJointPaperDimensions();
   }
 
+  private handleWindowDrag(): void {
+    const observables = getObservables(this.getJointPaper());
+    let ifMouseDown = false;
+    let ifMouseUp = false;
+    let down_offsetX = 0;
+    let down_offsetY = 0;
+    const elementOffset = this.getWrapperElementOffset();
+    // console.log('original offset: ', elementOffset.x, elementOffset.y);
+    // listen to the event when mouse was being clicked down.
+    this.getJointPaper().on('blank:pointerdown', function(evt: any, x: any, y: any) {
```

Comment:
Instead of calling `this.getJointPaper().on('blank:pointerdown'...`, make use of the Observable property provided by `rxjs` libary.

For instance, for the paper events listener, you can use 
` Observable.fromEvent<JointPaperEvent>(this.getJointPaper(), 'blank:pointerdown').subscribe()`
` Observable.fromEvent<JointPaperEvent>(this.getJointPaper(), 'blank:pointerup').subscribe()`


As for the mouse move event, you can use:

`Observable.fromEvent<MouseEvent>(document, 'mousemove')`